src/App.js

```diff
@@ -1,12 +1,15 @@
-import { useEffect, useRef, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
```

Comment:
since August of 2020 when was React v17.0 was released, we don't need to import React anymore for the JSX, more details:
https://www.chakshunyu.com/blog/why-did-and-dont-you-need-to-import-react/